Library/Formula/svmlight.rb

```diff
@@ -0,0 +1,16 @@
+class Svmlight < Formula
+  homepage "http://svmlight.joachims.org/"
+  url "http://download.joachims.org/svm_light/current/svm_light.tar.gz"
```

Comment:
Our concern is that this will break things when they update the version. Would it be possible to ask upstream for a versioned link? Thanks!